package.json

```diff
@@ -45,6 +47,7 @@
     "@babel/core": "^7.0.0",
     "@babel/preset-env": "^7.0.0",
     "babel-core": "^7.0.0-bridge.0",
+    "husky": "^0.14.3",
```

Comment:
please don't use husky; auto-installing git hooks is supremely dangerous and not worth the effort, since CI is the only reliable enforcement.